doc/source/ray-core/scheduling/accelerators.rst

```diff
@@ -0,0 +1,563 @@
+.. _gpu-support:
+.. _accelerator-support:
+
+Accelerator Support
+===================
+
+Accelerators (e.g. GPUs) are critical for many machine learning applications.
+Ray natively supports many accelerators as pre-defined :ref:`resource <core-resources>` types and allows tasks and actors to specify their accelerator :ref:`resource requirements <resource-requirements>`.
+
+The accelerators natively supported by Ray are:
```

Comment:
@jjyao please make it "Ray Core" since this is in the Core section. Train and Serve are on different timelines (and we should ideally have a high level definition of this...I'll take the AI on that).

cc @akshay-anyscale @matthewdeng